s3_website.yml

```diff
@@ -15,31 +15,32 @@ exclude_from_upload:
   - spelling
   - latex
 
-cloudfront_distribution_id: E1IDKKW65Q2HVZ
+cloudfront_distribution_id: E39A2ZMIIK7FK3
 
 cloudfront_distribution_config:
   default_cache_behavior:
     min_ttl: <%= 60 * 60 * 24 %>
   aliases:
     quantity: 1
     items:
-      - docs.opendatakit.org
+      - docs.getodk.org
 
 cloudfront_wildcard_invalidation: true
 
 # folder redirects. executed from top to bottom.
+# these redirects may not be necessary depending on how we redirect from docs.opendatakit.org
```

Comment:
Yes, I think we should do all of the odk2/odkx transformations only at docs.opendatakit.org and aim not to have any of those old prefixes that hit `docs.getodk.org`.

Reply:
So that means basically keeping an empty site (empty index.html) on docs.opendatakit.org, right. Probably set up to auto-deploy from a repo consisting just of an s3_website.yml on dfsg?

Reply:
Ah yes, I hadn't considered that this means maintaining and deploying something to that URL. I like that approach. @yanokwa?

Reply:
He likes that. So we'll have a tiny repo for this on github.com/dsfsg, and it can be tested whilst deployed on its cloudfront URL (before switching to the docs.opendatakit.org) domain.